### PR TITLE
fix(Hotkey): Support multiple modifiers

### LINKED ISSUE #1156: [Bug] V9, Ctrl+Shift+F2 not possible hotkey
Bug similar to #1150 

Previously, control + shift + F2 was possible in V8, and with what is written in electron documentation, it should be possible to do it, is it possible to fix it ?

# Preview

image

### PR DESCRIPTION
**Pull request by ke1v**
Fixes #1156

### CODE REVIEW

File: src/common/Core/Hotkey.test.ts [hunk 1]
```diff
@@ -30,12 +30,27 @@ describe(isValidHotkey, () => {
                 "Alt+]",
                 "Alt+(",
                 "Alt+)",
+                "Ctrl+Shift+Space",
+                "Alt+Shift+F2",
+                "Ctrl+Alt+Shift+Super+F8",
```
Comment (oliverschwendener) [inline src/common/Core/Hotkey.test.ts:35]: Are you shure that `Ctrl+Alt+Shift+Super+F8` is working?

Comment (oliverschwendener) [reply src/common/Core/Hotkey.test.ts:35]: See this article: https://www.electronjs.org/docs/latest/api/accelerator

Comment (ke1v) [reply src/common/Core/Hotkey.test.ts:35]: I tested it and it worked for me. I don't see anything from Electron docs that limit the number of modifiers keys?

Comment (oliverschwendener) [reply src/common/Core/Hotkey.test.ts:35]: True, just tested it myself. Somehow I thought this doesn't work.

**Review by oliverschwendener — APPROVED** [review]

**Comment by oliverschwendener** [comment]
Looks good, thank you for the efforts!

**Comment by oliverschwendener** [comment]
By the way, as you're now a collaborator on this repository, you can also create branches on this repo directly ;)

**Pull request merged**